`WebSocketOverHTTP2Test.testWebSocketConnectPortDoesNotExist()` is flaky

**Jetty version(s)**
12.0.x

**Description**
`org.eclipse.jetty.ee8.websocket.tests.WebSocketOverHTTP2Test.testWebSocketConnectPortDoesNotExist()` sometimes fails with:

```
java.lang.AssertionError: 
Expected: an instance of java.net.ConnectException
     but: <java.io.IOException: frame_size_error/invalid_frame_length> is a java.io.IOException
	at org.hamcrest.MatcherAssert.assertThat(MatcherAssert.java:20)
	at org.hamcrest.MatcherAssert.assertThat(MatcherAssert.java:6)
	at org.eclipse.jetty.ee8.websocket.tests.WebSocketOverHTTP2Test.testWebSocketConnectPortDoesNotExist(WebSocketOverHTTP2Test.java:224)
	at java.base/java.lang.reflect.Method.invoke(Method.java:568)
	at java.base/java.util.concurrent.RecursiveAction.exec(RecursiveAction.java:194)
	at java.base/java.util.concurrent.ForkJoinTask.doExec(ForkJoinTask.java:373)
	at java.base/java.util.concurrent.ForkJoinPool$WorkQueue.topLevelExec(ForkJoinPool.java:1182)
	at java.base/java.util.concurrent.ForkJoinPool.scan(ForkJoinPool.java:1655)
	at java.base/java.util.concurrent.ForkJoinPool.runWorker(ForkJoinPool.java:1622)
	at java.base/java.util.concurrent.ForkJoinWorkerThread.run(ForkJoinWorkerThread.java:165)
```

See: https://jenkins.webtide.net/blue/organizations/jenkins/jetty.project/detail/PR-10431/62/tests

--- a/jetty-ee10/jetty-ee10-websocket/jetty-ee10-websocket-jetty-tests/src/test/java/org/eclipse/jetty/ee10/websocket/tests/WebSocketOverHTTP2Test.java
+++ b/jetty-ee10/jetty-ee10-websocket/jetty-ee10-websocket-jetty-tests/src/test/java/org/eclipse/jetty/ee10/websocket/tests/WebSocketOverHTTP2Test.java
@@ -264,8 +264,11 @@ public class WebSocketOverHTTP2Test
         startServer();
         startClient(clientConnector -> new ClientConnectionFactoryOverHTTP2.HTTP2(new HTTP2Client(clientConnector)));
 
+        // Port 293 is not assigned by IANA, so
+        // it should be impossible to connect.
+        int nonExistingPort = 293;
         EventSocket wsEndPoint = new EventSocket();
-        URI uri = URI.create("ws://localhost:" + (connector.getLocalPort() + 1) + "/ws/echo");
+        URI uri = URI.create("ws://localhost:" + nonExistingPort + "/ws/echo");
 
         ExecutionException failure = Assertions.assertThrows(ExecutionException.class, () ->
             wsClient.connect(wsEndPoint, uri).get(5, TimeUnit.SECONDS));

--- a/jetty-ee9/jetty-ee9-websocket/jetty-ee9-websocket-jetty-tests/src/test/java/org/eclipse/jetty/ee9/websocket/tests/WebSocketOverHTTP2Test.java
+++ b/jetty-ee9/jetty-ee9-websocket/jetty-ee9-websocket-jetty-tests/src/test/java/org/eclipse/jetty/ee9/websocket/tests/WebSocketOverHTTP2Test.java
@@ -248,8 +248,11 @@ public class WebSocketOverHTTP2Test
         startServer();
         startClient(clientConnector -> new ClientConnectionFactoryOverHTTP2.HTTP2(new HTTP2Client(clientConnector)));
 
+        // Port 293 is not assigned by IANA, so
+        // it should be impossible to connect.
+        int nonExistingPort = 293;
         EventSocket wsEndPoint = new EventSocket();
-        URI uri = URI.create("ws://localhost:" + (connector.getLocalPort() + 1) + "/ws/echo");
+        URI uri = URI.create("ws://localhost:" + nonExistingPort + "/ws/echo");
 
         ExecutionException failure = Assertions.assertThrows(ExecutionException.class, () ->
             wsClient.connect(wsEndPoint, uri).get(5, TimeUnit.SECONDS));

